docs/shrc.md

```diff
@@ -66,43 +140,138 @@ The [-Xshareclasses](xshareclasses.md) option is highly configurable, allowing y
+        Cache name              level         cache-type      feature         layer       OS shmid       OS semid       last detach time
+
+        Compatible shared caches
+        Cache1                  Java8 64-bit  persistent      cr              0                                         Mon Sep 23 11:41:04 2019
+        ```
+
+3. The next Docker layer up is an Open Liberty image that is built on the OpenJ9 image. As part of this image, the `-Xshareclasses:name=Cache1,layer=1` suboption is used to create another cache called `Cache1`. Because the `layer=1` suboption is specified, this new cache is a layered cache, which builds on `Cache1` in the previous container layer. (Open Liberty starts two VMs, so if you instead use the `createLayer` suboption here, two layered caches are created, with layer numbers of 1 and 2.) Note that cache layers are different from, and independent of, container layers.
+
+4. In the same way, another Docker layer is added for an Open Liberty Java application, and another layered cache is created to add to `Cache1`. The `listAllCaches` suboption now shows all the caches and their layers:
+
+        ```
+        java -Xshareclasses:listAllCaches
+        ...
+        Cache name              level         cache-type      feature         layer       OS shmid       OS semid       last detach time
+
+        Compatible shared caches
+        Cache1                  Java8 64-bit  persistent      cr              0                                         Mon Sep 23 11:41:04 2019   
+        Cache1                  Java8 64-bit  persistent      cr              1                                         Mon Sep 23 11:46:25 2019
+        Cache1                  Java8 64-bit  persistent      cr              2                                         In use
+        ```  
+
+The caches are created in the same directory.
+
+When you use the `-Xshareclasses:name=Cache1` suboption in future Java commands, all the caches are started. The top-layer cache is started in read/write mode, and lower-layer caches are started in read-only mode. Modifying a lower-layer cache would invalidate all the caches in the layers above.
+
+The following options and cache utilities are available for creating, managing, and removing layered caches:
+
+- [`-Xshareclasses:createLayer`](xshareclasses.md#createlayer)
+- [`-Xshareclasses:layer`](xshareclasses.md#layer)
+- [`-Xshareclasses:printTopLayerStats`](xshareclasses.md#printtoplayerstats) (for example output, see [printTopLayerStats](shrc_diag_util.md#printtoplayerstats))
+- [`-Xshareclasses:destroyAllLayers`](xshareclasses.md#destroyalllayers)
+
+## Saving a shared classes cache
+
+As described in an earlier section, a shared classes cache can be persistent or non-persistent. Non-persistent caches are stored in shared memory and are removed when a system is rebooted. If you want to save a cache beyond a reboot you might want to consider making your cache persistent or taking a cache snapshot.
```

Comment:
Persistent caches are memory mapped files.

> If you want to save a cache beyond a reboot you might want to consider making your cache persistent

Seems obsolete, persistent caches are the default except on z/OS where persistent isn't an option.

Reply:
After " shared classes cache can be persistent or non-persistent." we can add "Persistent caches are memory mapped files.".